### DISCUSSION TITLE
Welcome to App-Waktu-Solat-Malaysia Discussions!

### DISCUSSION THREAD

**Discussion by iqfareez** [mptwaktusolat/app_waktu_solat_malaysia#36] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Muqrihafiy** [comment]
Assalam bro, maaf menganggu, boleh ke kalau saya nak tau cara apa yang bro guna utk sync waktu solat

**Reply by iqfareez** [reply]
Waalaikumsalam. Sorry, I overlook this post. We had some conversation on Instagram previously. Harap semuanya baik2 saja.
